pkg/config/load_test.go

```diff
@@ -34,20 +34,12 @@ func TestGetConfigShouldLoadFromCustomDir(t *testing.T) {
 
 	err = ioutil.WriteFile(path.Join(dir, "cog.yaml"), []byte(testConfig), 0o644)
 	require.NoError(t, err)
+	err = ioutil.WriteFile(path.Join(dir, "requirements.txt"), []byte("torch==1.0.0"), 0o644)
+	require.NoError(t, err)
 	conf, _, err := GetConfig(dir)
 	require.NoError(t, err)
-	want := &Config{
-		Predict: "predict.py:SomePredictor",
-		Build: &Build{
-			PythonVersion:      "3.8",
-			PythonRequirements: "requirements.txt",
-			SystemPackages: []string{
```

Comment:
Do we care about no longer checking `SystemPackages` here?

Reply:
I don't think so -- this test isn't testing this. I was intentionally making it less brittle.